tests/tests_api/test_student.py

```diff
@@ -132,7 +149,7 @@ def test_student_update(api_client, availability_slots):
     )
     Student.objects.get(pk=student.pk)
     fields_to_update = {
-        "project_status": StudentProjectStatus.BANNED,
+        "project_status": StudentProjectStatus.NEEDS_INTERVIEW_TO_DETERMINE_LEVEL,
```

Comment:
One thing that we need to remember is that if we update student project_status and language and level are empty then it will throw an error. So we kind of can proceed with a student only if it has language and level

Reply:
It is, yes. After the interview, the coordinator must edit the student's record in the dashboard and set their level